lib/mayaUsd/fileio/utils/meshReadUtils.cpp

```diff
@@ -58,6 +58,9 @@ TF_DEFINE_PUBLIC_TOKENS(UsdMayaMeshColorSetTokens,
 TF_DEFINE_PRIVATE_TOKENS(
     _meshTokens,
 
+    // Default UV set name in Maya
+    (map1)
```

Comment:
It might be nice to have this token available publicly in a place that's accessible to both this file and `meshWriteUtils.cpp` so we can avoid hard-coding the string there also:
https://github.com/Autodesk/maya-usd/blob/e504341c1cbf455653d9ff7c2c3033735cfca8d2/lib/mayaUsd/fileio/utils/meshWriteUtils.cpp#L838